client/src/components/TrailShowPage.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useEffect } from "react"
+
+const TrailShowPage = (props) => {
+  const [trail, setTrail] = useState({})
+
+  const getTrail = async () => {
+    const id = props.match.params.id
+    try {
+      const response = await fetch(`/api/v1/trails/${id}`)
+      if (!response.ok) {
+        const errorMessage = `${response.status} (${response.statusText})`
+        const error = new Error(errorMessage)
+        throw error
+      }
+      const body = await response.json()
+      setTrail(body.trail)
+    } catch (error) {
+      console.error(`Error in fetch: ${error.message}`)
+    }
+  }
+
+  useEffect(() => {
+    getTrail()
+  }, [])
+
+  return (
+    <div>
+      <ul>
+        <li>Name: {trail.trailName}</li>
+        <li>Length of Trail: {trail.trailLength}</li>
+        <li>Description: {trail.trailDescription}</li>
+        <li>Location: {trail.trailLocation}</li>
+        <li>Estimate Time: {trail.estimateTime}</li>
+      </ul>
```

Comment:
A few "best practice" but non-blocking comments here:

1. Heads up that if any of this information is optional, you're going to end up with list items that list the field, but no information! For optional info, it may be better to set up a variable above the `return` that then conditionally renders the whole `li` on the page:

```
let descriptionSection
if(trail.trailDescription) {
  descriptionSection = <li>Description: {trail.trailDescription}</li>
  }
```

2. Ideally, your field/attribute names would not be namespaced the way they are. E.g., your `trails` table would have `name`, `length`, `description`, and `location` columns, not `trailName`, `trailLength`, etc.

3. I'd love to see some variation in the HTML tags here: perhaps some headers, paragraphs, etc rather than a simple `ul`? Not a very engaging User Interface!